sources/install.sh

```diff
@@ -3228,6 +3228,13 @@ function install_PassTheCert() {
   add-test-command "passthecert.py --help"
 }
 
+function install_bqm() {
+  colorecho "Installing BQM"
+  gem install bqm --no-wrapper
+  add-history bqm
```

Comment:
history file missing in PR like #102 and #103, please take notice for future PRs